plugins/autocomplete/plugin.js

```diff
@@ -941,6 +941,13 @@
 				top = Math.max( rect.bottom, editorViewportRect.top );
 			}
 
+			var windowHeight = this.element.getWindow().$.innerHeight;
```

Comment:
I see you are still using `this.element.getWindow().$.innerHeight` and `this.element.getWindow().$.innerWidth` in your code instead of `getViewPaneSize()`. Is there any reason for that? Any edge cases which behaves differently? I see there are some slight differences (https://stackoverflow.com/a/7205786/646871) but we use it most cases so it should do the job :thinking: